docs/src/reference/the-traversal.asciidoc

```diff
@@ -2302,6 +2302,281 @@ link:++https://tinkerpop.apache.org/javadocs/x.y.z/core/org/apache/tinkerpop/gre
 link:++https://tinkerpop.apache.org/javadocs/x.y.z/core/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.html#mean-org.apache.tinkerpop.gremlin.process.traversal.Scope-++[`mean(Scope)`],
 link:++https://tinkerpop.apache.org/javadocs/x.y.z/core/org/apache/tinkerpop/gremlin/process/traversal/Scope.html++[`Scope`]
 
+
+[[mergeedge-step]]
+=== MergeEdge Step
+
+The `mergeE()` -step is used to add edges and their properties to a graph in a "create
+if not exist" fashion. The `mergeE` step can also be used to find edges matching a given
+pattern. This step was added to Gremlin as part of the TinkerPop 3.6 release. Prior to
+that, it was necessary to write more complex queries using steps such as `coalesce`,
```

Comment:
from a formatting perspective reference documentation refers to step names with double parens typically: so "`coalesce`" would be "`coalesce()`" or "`coalesce()`-step". Could you please review your content to fix that formatting a bit to keep things looking consistent?